Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -144,9 +144,13 @@ A specific will contain a specific release of Ceph as well as security fixes fro
     This setting only applies to new monitors that are created when the requested
     number of monitors increases, or when a monitor fails and is recreated. An
     [example CRD configuration is provided below](./pvc-cluster.md).
+    !!! note
+        This field should not be used if you are defining a specific `volumeClaimTemplate` for each zone in the `zones` section, as it will be overridden by the zone-specific configurations.
 * `failureDomainLabel`: The label that is expected on each node where the mons
     are expected to be deployed. The labels must be found in the list of
     well-known [topology labels](#osd-topology).
+    !!! note
+        This setting is an alternative to specifying a `topologySpreadConstraints` rule with the chosen label (e.g., `topology.kubernetes.io/zone`) for the mons under the `placement` section, as described in the [Placement Configuration Settings](#placement-configuration-settings).
```

Comment:
These notes must be preceded and followed by empty lines to make sure they format correctly.